src/Form/Register.php

```diff
@@ -25,6 +29,12 @@ protected function init(): void
         $this->buttonSave->set('Register');
         $this->buttonSave->addClass('large fluid');
         $this->buttonSave->iconRight = 'right arrow';
+
+        if ($this->linkSuccess !== [null]) {
```

Comment:
this is new, why?

Reply:
Sometimes after new user signup we want to redirect to some page instead of just showing success message and be stuck there. A bit similar like after login - we can redirect somewhere.